projects/js-packages/base-styles/composer.json

```
@@ -0,0 +1,30 @@
+{
+	"name": "automattic/jetpack-base-styles",
+	"description": "Jetpack Emerald Base Styles",
+	"type": "library",
+	"license": "GPL-2.0-or-later",
+	"require": {},
+	"require-dev": {
+		"yoast/phpunit-polyfills": "0.2.0",
+		"automattic/jetpack-changelogger": "^1.2"
+	},
+	"autoload": {
+		"classmap": [
+			"src/"
+		]
+	},
+	"scripts": {
+		"build-development": "echo 'Add your build step to composer.json, please!'"
+	},
+	"repositories": [
+		{
+			"type": "path",
+			"url": "../../packages/*",
+			"options": {
+				"monorepo": true
+			}
+		}
+	],
+	"minimum-stability": "dev",
+	"prefer-stable": true
```

Let's add the other monorepo tools here:

```suggestion
	"prefer-stable": true,
	"extra": {
		"autotagger": true,
		"npmjs-autopublish": true,
		"mirror-repo": "Automattic/jetpack-base-styles",
		"changelogger": {
			"link-template": "https://github.com/Automattic/jetpack-base-styles/compare/${old}...${new}"
		}
	}
```